EDDI/ChangeLog.md

```diff
@@ -2,6 +2,10 @@
 
 Full details of the variables available for each noted event, and VoiceAttack integrations, are available in the individual [event pages](https://github.com/EDCD/EDDI/wiki/Events).
 
+### 3.5.3
```

Comment:
I wasn't sure if I should add this in my PR - if it isn't necessary, I will remove it

Reply:
The change log is usually updated by the merger (to prevent merge conflicts). That said, as long as this PR is the only one writing to the change log then we'll still avoid merge conflicts. ;-)